### DISCUSSION TITLE
Slightly different color shades on every redraw

### DISCUSSION THREAD

**Discussion by yoav-lavi** [helix-editor/helix#8095] [Troubleshooting]
### Summary

<img width="402" alt="Screenshot 2023-08-28 at 15 25 31" src="https://github.com/helix-editor/helix/assets/14347895/03bda031-a82a-44ac-973d-b0d5290701ff">
<img width="348" alt="Screenshot 2023-08-28 at 15 25 26" src="https://github.com/helix-editor/helix/assets/14347895/5239f9a3-0049-4f85-994d-8d3f7cfe21f8">

I seem to get a slightly different shade of colors on every redraw of the terminal (toggling between two shades)

(for example, cyan here jumps between `#6FD1C2` and `#87CFC2`)

Attaching my theme

[tokyonightextra.toml.txt](https://github.com/helix-editor/helix/files/12454386/tokyonightextra.toml.txt)

I'm using the `wezterm` `.terminfo` if that's related.

I'm using `AppleFontSmoothing = 0;`

### Reproduction Steps

 

I tried this:

1. `hx .`
2. open a file
4. press down

I expected this to happen:

No color changes

Instead, this happened:

Color changes

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

Will add later if relevant

</details>

### Platform

macOS

### Terminal Emulator

wezterm 20230712-072601-f4abf8fd

### Helix Version

072e1ea

**Comment by yoav-lavi** [comment]
I can't seem to reproduce this anymore so it's likely something that happens under specific conditions

**Comment by pascalkuthe** [comment]
I doubt this is related to helix, likely something system-level instead.  I can't reproduce this (and neither can you now)
